relay-event-normalization/src/normalize/span/description/mod.rs

```diff
@@ -111,6 +113,17 @@ fn scrub_http(string: &str) -> Option<String> {
     Some(scrubbed)
 }
 
+fn scrub_file(description: &str) -> Option<String> {
+    let filename = match description.split_once(' ') {
+        Some((filename, _)) => filename,
+        _ => description,
+    };
+    match Path::new(filename).extension() {
+        Some(extension) => extension.to_os_string().into_string().ok(),
+        _ => None,
+    }
```

Comment:
I think this saves one allocation

```suggestion
    Some(Path::new(filename).extension()?.to_str()?.into_owned())
```